cmd/restic/cmd_backup.go

```diff
@@ -534,16 +534,18 @@ func runBackup(opts BackupOptions, gopts GlobalOptions, term *termstatus.Termina
 		targets = []string{filename}
 	}
 
-	sc := archiver.NewScanner(targetFS)
-	sc.SelectByName = selectByNameFilter
-	sc.Select = selectFilter
-	sc.Error = p.ScannerError
-	sc.Result = p.ReportTotal
+	if !quiet {
```

Comment:
Why not use ```gopts.Quiet``` and remove changes in definition of ```runBackup```?

Reply:
Oh, good catch, I've somehow missed that gopts is available inside the runBackup function. I've updated to commits accordingly.